dali/operators/image/convolution/gaussian_blur.h

```diff
@@ -50,6 +58,40 @@ class GaussianBlur : public Operator<Backend> {
   std::unique_ptr<OpImplBase<Backend>> impl_;
 };
 
+namespace gaussian_blur {
+
+constexpr static const char* kSigmaArgName = "sigma";
+constexpr static const char* kWindowSizeArgName = "window_size";
+
+template <int axes>
+inline GaussianBlurParams<axes> GetSampleParams(int sample, const OpSpec& spec,
```

Comment:
I was a bit surprised that this is here and not in the file that is literally called: `gaussian_blur_params.h`

Reply:
Agree, but in that file we only have the structure and Gaussian Window generation, and here we have a code related to the operator, that fills the parameters. Maybe some rename of this function?

Changed to ObtainSampleParams and added a docstring.